tests/jni_api.rs

```diff
@@ -572,22 +577,11 @@ pub fn throw_new() {
 
     let result = env.throw_new("java/lang/RuntimeException", "Test Exception");
     assert!(result.is_ok());
-    let ex: JObject = env
-        .exception_occurred()
-        .expect("Exception should be thrown")
-        .into();
-    assert_pending_java_exception(&env);
-
-    assert!(env
-        .is_instance_of(ex, "java/lang/RuntimeException")
-        .unwrap());
-    let message = env
-        .call_method(ex, "getMessage", "()Ljava/lang/String;", &[])
-        .unwrap()
-        .l()
-        .unwrap();
-    let msg_rust: String = env.get_string(message.into()).unwrap().into();
-    assert_eq!(msg_rust, "Test Exception");
+    assert_pending_java_exception_detailed(
+        &env,
+        Some("java/lang/RuntimeException"),
```

Comment:
Should `DEFAULT_EXCEPTION_TYPE` be used here?

Reply:
It's a different test and it is just a coincidence that it uses j.l.RuntimeException. Well... I see no reason making the `DEFAULT_EXCEPTION_CLASS` public. What if I define `RUNTIME_EXCEPTION_CLASS` (in analogue to the `EXCEPTION_CLASS` & `ARITHMETIC_EXCEPTION_CLASS`) and change all string representations to it (with no emphasis on default type of exception)?

Reply:
Implemented in the suggested way.